spec/system/discourse_login_client_spec.rb

```diff
@@ -24,47 +24,23 @@
 
   after { reset_omniauth_config(:discourse_login) }
 
-  let(:login_form) { PageObjects::Pages::Login.new }
   let(:signup_form) { PageObjects::Pages::Signup.new }
 
   context "when user does not exist" do
-    it "fills the signup form" do
+    it "skips the signup form and creates the account directly" do
```

Comment:
This spec fails now that we enforce `auth_skip_create_confirm = true`, right? I wonder if adding that to the plugin here is useful in the long run. Some client sites may want to keep it, for example, if they have extra custom user fields. (Food for thought for later.)